### PR TITLE
feat: add --fold-skipped=yes|no cli option

### PR DESCRIPTION
**Pull request by pbrezina**
This controlls how the skipped tests are displayed in the short summary.

- yes (default): keeps the current behavior that folds the skipped tests
  together
- no: each skipped test is on its own line, display as any other status

Resolves: https://github.com/pytest-dev/pytest/issues/9876

### CODE REVIEW

**Comment by pbrezina** [comment]
Please, let me know if you'd accept this change. If yes, I will include tests and documentation.

@nicoddemus

**Comment by nicoddemus** [comment]
I think it is reasonable, @The-Compiler @bluetech @RonnyPfannschmidt any objections?

File: src/_pytest/terminal.py [hunk 1]
```diff
@@ -154,6 +154,14 @@ def pytest_addoption(parser: Parser) -> None:
         dest="no_summary",
         help="Disable summary",
     )
+    group._addoption(
+        "--fold-skipped",
```
Comment (RonnyPfannschmidt) [inline src/_pytest/terminal.py:158]: if we don't plan to to add more variants, lets go with `--no-fold-skipped` as a store false, default true action following  the way flags work in tools like click

Comment (pbrezina) [reply src/_pytest/terminal.py]: Done.

**Comment by pbrezina** [comment]
Thank you. I added test, docs and changed option to --no-fold-skipped.

**Review by nicoddemus — APPROVED** [review]
Thanks @pbrezina! 

LGTM, left a minor suggestion.

**Comment by nicoddemus** [comment]
@RonnyPfannschmidt would you like to take another look before we squash/merge?

**Review by RonnyPfannschmidt — APPROVED** [review]

**Comment by nicoddemus** [comment]
Thanks @pbrezina!

**Pull request merged**